com.steamgriddb.steam-rom-manager.json

```diff
@@ -0,0 +1,135 @@
+				"install -Dm644 srm_48.png /app/share/icons/hicolor/48x48/apps/com.steamgriddb.steam-rom-manager.png",
+				"install -Dm644 srm_96.png /app/share/icons/hicolor/96x96/apps/com.steamgriddb.steam-rom-manager.png",
+				"install -Dm644 srm_256.png /app/share/icons/hicolor/256x256/apps/com.steamgriddb.steam-rom-manager.png",
+				"install -Dm644 srm_512.png /app/share/icons/hicolor/512x512/apps/com.steamgriddb.steam-rom-manager.png",
+				"install -Dm644 com.steamgriddb.steam-rom-manager.appdata.xml -t /app/share/metainfo",
+				"install -Dm644 com.steamgriddb.steam-rom-manager.desktop -t /app/share/applications",
+				"install -D apply_extra -t /app/bin",
+				"cp /usr/bin/ar /app/bin"
+			],
+			"sources": [
+				{
+					"type": "script",
+					"dest-filename": "apply_extra",
+					"commands": [
+						"ar x srm.deb",
+						"tar xf data.tar.xz",
+						"mv \"opt/Steam ROM Manager\" steam-rom-manager",
+						"rm -r srm.deb control.tar.* data.tar.xz debian-binary usr opt"
+					]
+				},
+				{
+					"type": "script",
+					"dest-filename": "run.sh",
+					"commands": [
+						"/app/bin/zypak-wrapper.sh /app/extra/steam-rom-manager/steam-rom-manager"
+					]
+				},
+				{
+					"type": "file",
+					"path": "com.steamgriddb.steam-rom-manager.appdata.xml"
+				},
+				{
+					"type": "file",
+					"path": "com.steamgriddb.steam-rom-manager.desktop"
+				},
+				{
+					"type": "file",
+					"dest-filename": "srm_16.png",
+					"url": "https://github.com/SteamGridDB/steam-rom-manager/raw/master/src/assets/icons/16x16.png",
+					"sha256": "1e0863c171b858b34d05cd4c35f28f8c647047c80fc01777c339b9fd56ec837c"
+				},
+				{
+					"type": "file",
+					"dest-filename": "srm_32.png",
+					"url": "https://github.com/SteamGridDB/steam-rom-manager/raw/master/src/assets/icons/32x32.png",
+					"sha256": "6868e84ea191375c38589413e187594c0215f2e02a3a162925cd3d3e811affc5"
+				},
+				{
+					"type": "file",
+					"dest-filename": "srm_48.png",
+					"url": "https://github.com/SteamGridDB/steam-rom-manager/raw/master/src/assets/icons/48x48.png",
+					"sha256": "06952b6111a45ccadc40a553c1963a07783f518b8fb85ed2bdee862fdd8dcd65"
+				},
+				{
+					"type": "file",
+					"dest-filename": "srm_96.png",
+					"url": "https://github.com/SteamGridDB/steam-rom-manager/raw/master/src/assets/icons/96x96.png",
+					"sha256": "3e96abc9f8e8852161e25c07f65fbbea17ff644e2377972b5440e640c964c745"
+				},
+				{
+					"type": "file",
+					"dest-filename": "srm_256.png",
+					"url": "https://github.com/SteamGridDB/steam-rom-manager/raw/master/src/assets/icons/256x256.png",
+					"sha256": "dfcef88c8e8b262f548c8d2799ff1f13e688bd0f710b5afdf780f9fc2686243b"
+				},
+				{
+					"type": "file",
+					"dest-filename": "srm_512.png",
+					"url": "https://github.com/SteamGridDB/steam-rom-manager/raw/master/src/assets/icons/512x512.png",
+					"sha256": "371dbd9c9dbee38018ad32cff823660c91407f6b23ac7eec7d8860f5f271daaa"
+				},
+				{
+					"type": "extra-data",
```

Comment:
since this is GPL licensed software, do you really need to make this extra-data?

Reply:
technically no

Reply:
i can make it not extra if you like :)